students/ChrisH/lesson06/.gitignore

```diff
@@ -0,0 +1 @@
+.pytest_cache/
```

Comment:
adding this only works if you are adding to the root of the repo i think, not sure though.